### PR TITLE
More typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

File: orocos_kdl/doc/tex/UserManual.tex [hunk 1]
```diff
@@ -101,7 +101,7 @@ \subsection{Vector}
 
 \paragraph{Get/Set individual elements}
 \label{sec:gets-indiv-elem}
-The operators \lstinline$[]$ and \lstinline$()$ use indeces from 0..2,
+The operators \lstinline$[]$ and \lstinline$()$ use indices from 0..2,
```
Comment (luzpaz) [inline orocos_kdl/doc/tex/UserManual.tex]: should this be `indexes` instead ?

**Comment by luzpaz** [comment]
please review

**Comment by luzpaz** [comment]
@meyerj do you mind reviewing when you have a moment? Thanks!

File: orocos_kdl/doc/tex/UserManual.tex (hunk 1 above)
Comment (meyerj) [reply orocos_kdl/doc/tex/UserManual.tex]: Both would be fine, I guess.

**Pull request merged**